teamMembers.json

```diff
@@ -0,0 +1,338 @@
+[
+  {
+    "name": "Adriana Cionca",
+    "avatar": "./assets/contributors/adriana-cionca.webp",
```

Comment:
I feel like for some people the avatar is way to high quality and it looks weird. For example for almost all members in FRONTEND Team. Is there something you can do about this?

Reply:
I will look into it